stable/xray/README.md

```diff
@@ -47,6 +47,16 @@ To upgrade an existing Xray, you still use **helm**
 helm upgrade --set common.xrayVersion=2.1.2 jfrog/xray
 ```
 
+If xray was installed without providing a value to postgresql.postgresPassword (a password was autogenerated), follow these instructions:
+1. Get the current password by running:
+```bash
+POSTGRES_PASSWORD=$(kubectl get secret -n <namespace> <myrelease>-postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
```

Comment:
Need to also extract rabbitMQ and mongoDB passwords

Reply:
Done